### PR TITLE
Externalise the Transaction enableLogRetries property

### LINKED ISSUE #19976: Externalise enableLogRetries property
When recovery logs are stored in an RDBMS table, this property allows SQL Operations that fail to be retried.

### PR DESCRIPTION
**Pull request by neilgsyoung**
fixes #19976

### CODE REVIEW

**Comment by neilgsyoung** [comment]
#build

**Review by jonhawkes — APPROVED** [review]

File: dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -93,3 +93,7 @@ lease.length.desc=The length of time before a recovery log lease expires
 
 lease.renewal.threshold=Lease renewal threshold
 lease.renewal.threshold.desc=The percentage of the duration of the recovery log lease that passes before the lease is renewed
+
+enable.log.retries=Enable retries on database transaction logs
+enable.log.retries.desc=When recovery logs are stored in an RDBMS table, this property allows SQL Operations that fail to be retried.
```
Comment (dmuelle) [inline dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties]: looks good except I don't think "operations" needs to be capitalized 

When recovery logs are stored in an RDBMS table, this property allows SQL operations that fail to be retried.

Comment (neilgsyoung) [reply dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties]: Thanks, I've updated the metatype properties in a new commit.

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Comment by neilgsyoung** [comment]
#run-libby-bot

**Pull request merged**